.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: Feature request
```

Comment:
ISSUE: Inconsistent capitalization.

We use a capital H for Help, but use lowercase R for report and request.

- Need Help?
- Bug report
- Feature request

We should be consistent.